LootLockerServerSDK/Source/LootLockerServerSDK/Public/LootLockerServerForBlueprints.h

```diff
@@ -940,6 +941,62 @@ class LOOTLOCKERSERVERSDK_API ULootLockerServerForBlueprints : public UObject
     UFUNCTION(BlueprintCallable, Category = "LootLockerServer Methods | Purchases", meta = (DeprecatedFunction, DeprecationMessage = "This method has been deprecated. We will soon release an updated purchase system, so please refrain from using this method."))
     static void CheckPurchaseStatusForPlayerByPlatformTransactionID_IncludeProducts(int PlayerID, int PlatformTransactionID, const FLootLockerServerPurchaseStatusWithProductsResponseBP& OnCompletedRequest);
 
+    //==================================================
+    // Game Progressions https://ref.lootlocker.com/server-api/#progressions
+    //==================================================
+
+    /**
+     * Get a list of progressions for the game
+     * https://ref.lootlocker.com/server-api/#get-all-progressions
```

Comment:
Same comment as with the other progressions, add something about the amount getting returned.